schema/neighbourhood_schema_1-0-0.json

```diff
@@ -0,0 +1,199 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://Orange-OpenSource.github.io/its-client/schema/neighbourhood",
+  "description": "Neighbourhood JSON schema",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "message_type",
+    "version",
+    "neighbours"
+  ],
+  "properties": {
+    "message_type": {
+       "description" : "The type of schema for this object",
+       "type": "string",
+       "const": "neighbourhood"
+    },
+    "version": {
+      "description": "The version of the schema for this object",
+      "type": "string",
+      "const": "1.0.0"
+    },
+    "neighbours": {
+      "description": "The list of neighbours of an instance",
+      "type": "array",
+      "items": {
+        "oneOf": [
+          { "$ref": "#/$defs/mqtt_neighbour" }
+        ]
+      }
+    }
+  },
+  "$defs": {
+    "basic_neighbour": {
+      "type": "object",
+      "required": [
+        "neighbour_type",
+        "border_definition"
```

Comment:
why not to add the telemetry connection to gather the 2 connection tpes (message and telemetry) on one message? Otherwise, the telemetry connection elements must be gather into the information message on the neighbour.